packages/react-component-tree/test/unit/load-child.js

```diff
@@ -12,9 +12,9 @@ const StatelessComponent = {
 };
 
 describe('UNIT Load child', () => {
-  const FirstComponent = _.cloneDeep(ReactComponent);
-  const SecondComponent = _.cloneDeep(ReactComponent);
-  const ThirdComponent = _.cloneDeep(StatelessComponent);
+  const FirstComponent = cloneDeep(ReactComponent);
+  const SecondComponent = cloneDeep(ReactComponent);
+  const ThirdComponent = cloneDeep(StatelessComponent);
```

Comment:
This is odd, what must've I been thinking. Can you duplicate `ReactComponent` and remove these `cloneDeep` calls along with the peerDependency?